csvcubed/csvcubed/utils/csvdataset.py

```diff
@@ -0,0 +1,75 @@
+"""
+CSV Dataset
+-----------
+
+Utilities for CSV Datasets
+"""
+
+from enum import Enum
+from pathlib import Path
+from typing import List, Optional, Tuple
+from uuid import uuid1
+import pandas as pd
+
+from rdflib import Graph
+
+from csvcubed.cli.inspect.inspectsparqlmanager import select_single_unit_from_dsd
+from csvcubed.models.inspectsparqlresults import QubeComponentResult
+from csvcubed.cli.inspect.inspectdatasetmanager import (
+    get_measure_col_name_from_dsd,
+    get_single_measure_from_dsd,
+    get_unit_col_name_from_dsd,
+)
+
+
+class CanonicalShapeRequiredCols(Enum):
```

Comment:
Do we need this anymore? Can we just put `measure` and `unit` in the relevant functions which add the measures/units columns?

It feels a bit misleading to suggest that columns with these names are required in a canonically shaped cube.